1-exercises/D-array-filter/exercise.js

```diff
@@ -8,7 +8,9 @@
 
 let pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-let pairsByIndex; // Complete this statement
+let pairsByIndex = pairsByIndexRaw.filter(
+  (val) => Array.isArray(val) && val.length === 2
```

Comment:
Small detail, when an arrow function only has one argument you don't need the brackets: `...filter(val => Array(...`